R/dependencies.R

```diff
@@ -1641,10 +1638,8 @@ renv_dependencies_report <- function(errors) {
 
   # emit messages
   enumerate(splat, function(file, problem) {
-    lines <- paste(rep.int("-", nchar(file)), collapse = "")
-    prefix <- format(paste("ERROR", seq_along(problem$message)))
-    messages <- paste(prefix, problem$message, sep = ": ", collapse = "\n\n")
-    text <- c(file, lines, "", messages, "")
+    messages <- paste("Error", problem$message, sep = ": ", collapse = "\n\n")
```

Comment:
Since I was in here, I tweaked the formatting a little to match the style elsewhere.